packages/mobile/src/screens/profile-screen/ProfileHeader/TopSupporters.tsx

```diff
@@ -97,6 +98,14 @@ export const TopSupporters = () => {
     })
   }, [navigation, user_id])
 
+  useLayoutEffect(() => {
+    // Prevent multiple re-renders from rankedSupporters changing
+    if (rankedSupporters.length > 0 && !hasRankedSupportersLoaded.current) {
```

Comment:
nitty nit: could just do `rankedSupporters.length` instead of `rankedSupporters.length > 0`